labelbox/data/annotation_types/collection.py

```diff
@@ -179,10 +181,8 @@ def _apply_threaded(self, fns, max_concurrency, *args):
 
 class LabelGenerator(PrefetchGenerator):
     """
-    A container for interacting with a collection of labels.
-
-    Use this class if you have larger data. It is slightly harder to work with
-    than the LabelList but will be much more memory efficient.
+    A container for interacting with a large collection of labels. 
+    For a small number of labels, just use a list of Label objects.
     """
 
     def __init__(self, data: Generator[Label, None, None], *args, **kwargs):
```

Comment:
```
def as_list(self) -> "LabelList":
        warnings.warn("This method is deprecated and will be "
                      "removed in a future release. LabeList"
                      " class will be deprecated.")
```
Could you fix the spelling on `LabeList`?

Reply:
@kkim-labelbox done